src/entities/purchase-flow-params.ts

```diff
@@ -0,0 +1,24 @@
+import { type Package, type PurchaseOption } from "./offerings";
+
+/**
+ * Parameters used to customise the purchase flow when invoking the `.purchase` method.
+ * @public
+ */
+export interface PurchaseFlowParams {
```

Comment:
Another consistency suggestion 😅: what about using the purchases-android name, `PurchaseParams`? I think it would make context-switching between the various SDKs easier if concepts are named similarly.